README.md

```diff
@@ -1 +1,37 @@
-# hocs-bulk-case-action
\ No newline at end of file
+# Hocs-Bulk-Case-Action
+
+Hocs-Bulk-Case-Action is a Java application which interacts with [hocs-workflow](https://github.com/UKHomeOffice/hocs-workflow)
+to perform actions on cases in HOCS (Home Office Correspondence Service) in bulk.
+
+## Features
+### Close Case
+Currently the only service implemented which allows the application
+to close a list of cases which should be given as UUIDs.
```

Comment:
Does this list have to be in a specific format? Is it a CSV (with/without headers)?

Reply:
I've expanded this to add this information